lms/templates/courseware/course_about.html

```diff
@@ -105,6 +105,8 @@
 <%block name="pagetitle">${course.display_name_with_default_escaped}</%block>
 
 <section class="course-info">
+  <%block name="course_info_before_header"></%block>
+  <%block name="header">
```

Comment:
We will have to be careful with the naming. The `main.html` template (which this template inherits from)` also (indirectly) contains a `<header>` element. It is currently not wrapped in a named block, but it could be in the future and would clash with the `header` block from this template. We should name this one something like `course_about_header`.

Reply:
Maybe it would make sense to prefix all blocks introduced by a template by the template name.

Reply:
I renamed blocks to use `course_about` prefix.